pyTST/core.py

```diff
@@ -170,7 +189,8 @@ def import_from_txt(self, filename):
         self.u95_array = timedata[:, 1]
         self.mean_array = timedata[:, 2]
 
-    def plot(self, filename=None, show_cursor=True):
+    def plot(self, filename=None, fileFormat='.png',show_cursor=True,\
+            selectTimeDiscard=True,step_size=10):
```

Comment:
try to be consistent with the naming convention, ie: here don't use camel case since everything else is snake case (see https://www.python.org/dev/peps/pep-0008)
and with this it is no longer a filename but only a prefix

Reply:
I never heard about the terms camel vs snake case. Good point